src/main/java/com/foo/gosucatcher/domain/review/domain/Review.java

```diff
@@ -46,24 +47,32 @@ public class Review extends BaseEntity {
 	@JoinColumn(name = "sub_item_id")
 	private SubItem subItem;
 
-	private String description;
+	@OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.REMOVE, orphanRemoval = true)
+	@JoinColumn(name = "reply_id")
+	private Reply reply;
+
+	private String content;
 
 	private int rating;
 
 	private boolean isDeleted = false;
```

Comment:
이것두 머지 반영이 안된건가용? [벨덩](https://www.baeldung.com/spring-jpa-soft-delete) 에서도 아래 방법을 추천해네요!
```suggestion
	private boolean isDeleted = Boolean.FALSE;
```
객체 타입으로!

Reply:
아 저는 필드 초기화만 말씀하시는 줄 알고!! 수정하겠습니다~